packages/provider-url-service/src/index.ts

```diff
@@ -58,7 +59,7 @@ export const composeCicoProviderUrl = functions.https.onRequest((request, respon
         &hostLogoUrl=${VALORA_LOGO_URL}
         &fiatCurrency=${fiatCurrency}
         &fiatValue=${fiatAmount}
-        &finalUrl=${encodeURIComponent(CASH_IN_SUCCESS_DEEPLINK)}
+        &finalUrl=${encodeURIComponent(cashInSuccessDeepLink)}
```

Comment:
Are we missing Transak and Simplex or do they not use the cash in success screen?

Reply:
Simplex redirect will be possible only once the API integration goes live. Transak doesn't allow deeplink redirects for some reason.